.gitignore

```diff
@@ -8,6 +8,7 @@ backups/**
 staging/**
 logs/**
 .DS_Store
+setting.json
```

Comment:
i think this is a vs code config file - gary must have added it, and I dont think its a problem